src/git/log.js

```diff
@@ -11,6 +11,7 @@ function log(repoPath, done) {
     cwd: repoPath
   }, function(error, stdout, stderr) {
     if (error) {
+      throw new Error(error);
```

Comment:
also, this makes the line below (`done();`) unreachable since a `throw` interrupts the execution of the function